src/pages/AssetPage/index.tsx

```diff
@@ -155,17 +164,26 @@ export class AssetPage extends React.Component<AssetPageProps> {
                     )}
                 </ul>
               )}
-              {this.isAvailableForCryptoWithdraw && (
-                <ul className="action-list">
-                  <li className="action-list__title">Withdraw</li>
-                  {this.renderMenuItem(
-                    ROUTE_WITHDRAW_CRYPTO_FROM(asset.id),
-                    `${process.env
-                      .PUBLIC_URL}/images/paymentMethods/deposit-bl-transfer-icn.svg`,
-                    'Blockchain Transfer'
-                  )}
-                </ul>
-              )}
+              {this.isAvailableForCryptoWithdraw ||
```

Comment:
true || (false && 'some text') will return true, so it mean that this code will render true in case this.isAvailableForCryptoWithdraw is true?

Reply:
Fixed